foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,58 @@
 
-### Quick exercises before starting
-
-1.  Identify three ways to include JavaScript in a page.
-2.  Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browser's console.
+<div class="lesson-note">
+<h4>Student solutions</h4>
+Some students' solutions below include buttons, text, and other elements. Those elements form what is known as a Graphical User Interface (GUI). You'll add those elements, the GUI, in a later lesson. So remember to push your code to GitHub.
+</div>
 
-Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving. Plan your solution out before writing any code, and test each piece as you build to ensure it is working before moving on to the next!
+### Quick exercise before starting
 
-### Assignment
+1. Identify three ways to include JavaScript in a page.
+2. Check if the page includes JavaScript:
+   - Write `console.log("Hello World");` in JavaScript.
+   - Check if "Hello World" is logged in the browser console once you open your webpage.
 
-<div class="lesson-content__panel" markdown="1">
-Don't forget to commit early & often! You can [reference the Commit Message lesson here](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages)!
+### Problem solving approach
 
-1.  Start a new Git repo for your project.
-2.  Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file).  This game is going to be played completely from the console, so don't worry about putting anything else in there.
-3.  Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'.  We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-4.  Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
-    *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
+Since this is your first JavaScript program built from scratch, remember the [previous wise words from the problem-solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving). Before diving into this project, first plan or pseudocode your solution, then write the code, then ensure the code works.
```

Comment:
[Proofreading] The last sentence seems to describe some steps, so I provided a suggestion on how to make it clearer. 
```suggestion
Since this is your first JavaScript program built from scratch, remember the [previous wise words from the problem-solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving). Before diving into this project, First, plan or pseudocode your solution. Next, write the code. Lastly, ensure the code works.
```

Reply:
Thank you, we are still in the exploratory phase it seems so no final version to review yet.